src/process/index.js

```diff
@@ -47,12 +51,39 @@ export default class ProcessServer {
         toCompare.push(p.replace('.x64', ''));
         toCompare.push(p.replace('x64', ''));
         toCompare.push(p.replace('_64', ''));
+        toCompare.push(p.replace('.exe', ''));
+      }
+
+      // remove duplicates to process
+      const UniqueToCompare = []
+      for (const u of toCompare) {
+        if (!UniqueToCompare.includes(u)) {
+          UniqueToCompare.push(u)
+        }
+      }
+      
+      if (UniqueToCompare.length < 1) { // Skip empty path
+        continue
       }
 
       for (const { executables, id, name } of DetectableDB) {
         if (executables?.some(x => {
           if (x.is_launcher) return false;
-          if (x.name[0] === '>' ? x.name.substring(1) !== toCompare[0] : !toCompare.some(y => x.name === y)) return false;
+          // I can't figure how to not make 'sbin/init' NOT match 'Divinity Original Sin' with a substring match, so this ugly hack have to do for now
```

Comment:
I'm not really happy with this code, but it works. I'm new to javascript/nodejs and I couldn't get a more elegant solution to work as I cannot wrap my brain around the arrow operator